src/docs/forms/components/CalendarInput/examples/02-ClearableCalendarInput.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import {CalendarInput} from "doj-react-adminlte/Form";
+
+export default class ClearableCalendarInput extends React.Component {
+    constructor(props) {
+        super(props);
+
+        this.state = {
+            clearableTrue: null,
+            clearableFalse: null
+        };
+    }
+
+    handleDateChange = (name, value) => {
+        this.setState({[name]: value});
+    };
+
+    render() {
+        const {clearableTrue,clearableFalse} = this.state;
```

Comment:
Instead of showing two `CalendarInput`s, just show one and add a CheckBox which will toggle the clearable flag of this component. Also, add an external button to demonstrate how to clear the checkbox using an external control. You can do this by creating a `handleClearClick` callback that will set the date state to `null` when clicked. This will allow us to demonstrate that the calendar input clear can still be triggered by an external source.

Reply:
Partially resolved as of 0922 30 July. Checkbox not yet implemented.